compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUEnableAArch64SSVE.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+//===- LLVMCPUEnableAArch64SSVE.cpp ---------------------------------------===//
+//
+// This pass enables the Armv9 Scalable Matrix Extension (SME) Streaming SVE
+// (SSVE) mode [1].
```

Comment:
Should we move this to the pass class and use `///` so that it's picked it up by Doxygen?